<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BEMUser_Interface</class>
 <widget class="QWidget" name="BEMUser_Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>886</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>170</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>170</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>120</green>
        <blue>215</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Garamond</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>BEMUser_Interface</string>
  </property>
  <property name="styleSheet">
   <string notr="true">colour = blue;</string>
  </property>
  <widget class="QPushButton" name="Button_Grid">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
border-color: grey;
border-width: 3px;
border-radius: 8px;</string>
   </property>
   <property name="text">
    <string>Templates</string>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="Output_Label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>840</y>
     <width>871</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Welcome to BEMUse! Go ahead and create your geometry.</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="GeometryBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>151</width>
     <height>471</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Step 1: Geometry</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="SolverBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Step 2: Solver</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="ExecuteBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>660</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Step 3: Calculate</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="Button_Execute">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Garamond</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
border-color: grey;
border-width: 3px;
border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Execute 
 Solver</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="NormVis">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>110</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Surface normals</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="SurfPot">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>130</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Radiation potential</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="WaveVis">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>170</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Radiated free surface</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>21</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Visualisation Options</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QCheckBox" name="WaveVis_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Garamond</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Scattered free surface</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="ExportButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
border-color: grey;
border-width: 3px;
border-radius: 8px;</string>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ImportButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
border-color: grey;
border-width: 3px;
border-radius: 8px;</string>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GeoDelete">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
border-color: grey;
border-width: 3px;
border-radius: 8px;</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Configure">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
border-color: grey;
border-width: 3px;
border-radius: 8px;</string>
   </property>
   <property name="text">
    <string>Configure 
 Solver</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="BoundaryVis">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>50</y>
     <width>161</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Geometry</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="IFSVis">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>70</y>
     <width>201</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Interior free surface</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="EFSVis">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>90</y>
     <width>201</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Exterior free surface</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="SurfDiff">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>150</y>
     <width>211</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Diffraction potential</string>
   </property>
  </widget>
  <zorder>GeometryBox</zorder>
  <zorder>groupBox_5</zorder>
  <zorder>Button_Grid</zorder>
  <zorder>Output_Label</zorder>
  <zorder>SolverBox</zorder>
  <zorder>ExecuteBox</zorder>
  <zorder>NormVis</zorder>
  <zorder>SurfPot</zorder>
  <zorder>WaveVis</zorder>
  <zorder>ExportButton</zorder>
  <zorder>ImportButton</zorder>
  <zorder>GeoDelete</zorder>
  <zorder>Configure</zorder>
  <zorder>BoundaryVis</zorder>
  <zorder>IFSVis</zorder>
  <zorder>EFSVis</zorder>
  <zorder>SurfDiff</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
